ACMILabs/interactive-label
# #7 Ignore playlist items with no label

*Resolves issue #7*

Ignore playlist items that don't have a label defined.

### Acceptance Criteria
- [x] Filter the playlist api response to include only playlist items that have labels defined.

### Relevant design files
* None

### Testing instructions
1. Build the development image: `cd development` and `docker-compose up --build`
1. Run the tests `docker exec -it labelinteractive make linttest`
1. Edit your `config.env` to point to `XOS_PLAYLIST_ID=16`
1. Restart docker `docker-compose up`
1. See that no no labels render: http://localhost:8081

### DoD
For requester to complete:
